Makefile

```
@@ -24,11 +24,17 @@ fmt-check:
 ci: fmt-check test
 
 OPA_VER=v0.39.0
-OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/opa_linux_amd64_static
+OPA_FILE=opa_linux_amd64_static
+OPA_URL=https://openpolicyagent.org/downloads/$(OPA_VER)/$(OPA_FILE)
+OPA_SHA_FILE=$(OPA_FILE).sha256
+OPA_SHA_URL=$(OPA_URL).sha256
 OPA_DEST=/usr/bin/opa
```

Perhaps like this so we don't need to `sudo`?
```suggestion
OPA_DEST=$HOME/opa
```